### ISSUE TITLE
About Raise RuntimeError while training with one singal GPU

### ISSUE THREAD

**Issue by Kimsure** [psychopa4/OVSR#3]
Thanks for your impressive work! However, There are some problems as I train the model with train.sh    
Error As follows
"Raise RuntimeWarning(f'epoch {epoch}, iteration {iteration}, loss {loss_v}')
RuntimeWarning: epoch 1, iteration 0, loss nan"
I dont know which reason cause the problem. the DDP or others?

**Comment by Kimsure** [comment]
I also find that after 1 epoch , the loss and PSNR are "-101.2321321..." and "-28.365432...". Why are they negative number? If they are positive number i think it will be true.

**Comment by Kimsure** [comment]
OK I have solved it. The reason is when I decrease the batch_size , the learning rate seems too large. Decrease the lr is ok.
